freqtrade/exchange/gateio.py

```diff
@@ -31,7 +31,7 @@ class Gateio(Exchange):
         # TradingMode.SPOT always supported and not required in this list
         # (TradingMode.MARGIN, MarginMode.CROSS),
         # (TradingMode.FUTURES, MarginMode.CROSS),
-        (TradingMode.FUTURES, MarginMode.ISOLATED)
+        # (TradingMode.FUTURES, MarginMode.ISOLATED)
```

Comment:
`gateio.fetchLeverageTiers` isn't implemented yet, but I just figured out how to implement it, and it will probably be merged today or tommorow

Reply:
i did do some futures trading for gate.io

leverageTiers may be required in the end to get details right - but to test with smallish stake i don't see why we couldn't (unless this PR breaks gate.io again - which i didn't really test).

Reply:
> i did do some futures trading for gate.io
> 
> leverageTiers may be required in the end to get details right - but to test with smallish stake i don't see why we couldn't (unless this PR breaks gate.io again - which i didn't really test).

I added the version of `get_max_leverage` and `get_maintenance_ratio_and_amt` that work for gateio as gateio methods, so that they override the version on exchange.py. Unless someone plans to trade over 10,000 at 100x leverage it should be accurate actually